packages/fxa-auth-server/lib/senders/emails/templates/postVerify/index.ts

```diff
@@ -8,14 +8,14 @@ export const render = () => `
   <mj-include path="./lib/senders/emails/css/global.css" type="css" css-inline="inline" />
    <mj-section>
      <mj-column>
-       <mj-text css-class="primary-text"><span data-l10n-id="postVerify-sub-title">Firefox Account verified. You're almost there.</span></mj-text>
-       <mj-text css-class="header-text"><span data-l10n-id="postVerify-title">Next sync between your devices!</span></mj-text>
+       <mj-text css-class="text-sub-body"><span data-l10n-id="postVerify-sub-title">Firefox Account verified. You're almost there.</span></mj-text>
```

Comment:
While refactoring the styles, I observed that the new template (left side) has a little more margin at the bottom after the first text section
![image](https://user-images.githubusercontent.com/37609519/130256426-61357c20-6cb2-4a0d-8d4f-a851b9806a2b.png)
We kinda missed it in the `postVerify` branch, but the new styles have taken care of that too